seriously-simple-stats.php

```diff
@@ -27,7 +27,7 @@
 
 use SeriouslySimpleStats\Classes\Stats;
 
-require_once 'vendor/autoload.php';
+require_once __DIR__ . 'vendor/autoload.php';
```

Comment:
@thomas-gordon thanks for this. I was not aware this would cause issues on Docker environments.

Could I ask you to try using a predefined constant instead, the same way we do in the Seriously Simple Podcasting plugin


define( 'SSS_PLUGIN_PATH', plugin_dir_path( __FILE__ ) );

require SSP_PLUGIN_PATH . 'vendor/autoload.php';

That would also make it compatible with Linux symlinks, so if it works on Docker, it solves both environments.

Reply:
:wave: have you been able to test this?

Reply:
I know this was a long time ago but I did this change in my docker environment and it solved the issue in our containers. We are using version `1.2.10` of the SSS plugin but I see the issue persisting with version `1.3.0`.

```
define( 'SSS_PLUGIN_PATH', plugin_dir_path( __FILE__ ) );

require_once SSS_PLUGIN_PATH . '/vendor/autoload.php';
```

Before this change, we were seeing this error
```
Fatal error: Uncaught Error: Class 'SeriouslySimpleStats\Classes\Stats' not found in /var/www/html/test-site/public/wp-content/plugins/seriously-simple-stats/seriously-simple-stats.php:48
```